templates/dev-chrisbanes-snapper/External-Dependency-Info.txt

```diff
@@ -0,0 +1,215 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
+Do not translate or localize
+
+Xamarin Components for Snapper incorporates 
+third party material from the projects listed below. The original copyright 
+notice and the license under which Microsoft received such third party 
+material are set forth below.  Microsoft reserves all other rights not 
+expressly granted, whether by implication, estoppel or otherwise.
+
+########################################
+# Napier
```

Comment:
Should this be "Snapper" instead of "Napier"?

Reply:
correct. I missed that one.

Fixed